spring-cloud-skipper-platform-cloudfoundry/src/main/java/org/springframework/cloud/skipper/deployer/CloudFoundryPlatformAutoConfiguration.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.skipper.deployer;
```

Comment:
When we split into new packages, should have unique package name as now we have:
```
spring-cloud-skipper-platform-cloudfoundry/src/main/java/org/springframework/cloud/skipper/deployer/CloudFoundryPlatformAutoConfiguration.java
spring-cloud-skipper-platform-kubernetes/src/main/java/org/springframework/cloud/skipper/deployer/KubernetesPlatformAutoConfiguration.java
```
Maybe `org.springframework.cloud.skipper.deployer.kubernetes` and `org.springframework.cloud.skipper.deployer.cloudfoundry`